docker/mnt/run.sh

```diff
@@ -0,0 +1,121 @@
+#! /bin/bash
+set -e
+
+if [ -z "$1" ]; then
+    # the main container need not stay running for services
+    exit 0
+fi
+
+function setup() {
+    [ -n "$1" ] && TEST="$1"
+
+    if [[ "$TEST" =~ ^python ]]; then
+
+        scripts/uninstall-requirements.sh
+        pip install \
+            -r requirements/requirements.txt \
+            -r requirements/dev-requirements.txt \
+            coveralls
```

Comment:
I've thought quite a bit about different options for this.
- One option is to put the virtualenv in a cached directory on the host and mount it into the container. That way it will not need to be rebuilt from scratch each time. I like this option best, but decided to defer implementing it until later because it's a little tricky to get the volume setup correctly.
- Currently since the dockerfile extends `dimagi/commcarehq_base:latest` there is already a not-too-outdated set of requirements installed when the container starts, so this doesn't take too long. This seemed like a reasonable stop-gap in the short term. Also, it might be a reasonable long-term solution if [`dimagi/commcarehq_base`](https://hub.docker.com/r/dimagi/commcarehq_base/) gets updated fairly frequently.